lms/db/__init__.py

```diff
@@ -113,6 +114,22 @@ def init(engine, drop=False, stamp=True):  # pragma: nocover
         if stamp:
             alembic.command.stamp(alembic.config.Config("conf/alembic.ini"), "head")
 
+        if data:
+            create_data(engine)
+
+
+def create_data(engine):
+    """
+    Create any data needed for the application.
+
+    This type of data is usually crated by migrations which are not executed
+    while using `init` above
+
+    Creating it here allows the data to available in the development environment and tests.
```

Comment:
This is the main reason to do this. In general we don't run the migration to start a new environment.

An alternative could be to move this to devdata but then will have to deal with this in unit/func/bdd tests anyway. 

I think there's an argument that this is fundamental data for the app that happends to live in the DB while devdata is just useful for testing in the local env.